typescript/src/bitcoin.ts

```diff
@@ -374,6 +481,26 @@ export function computeHash160(text: string): string {
   return hash160.digest(Buffer.from(text, "hex")).toString("hex")
 }
 
+/**
+ * Computes the double SHA256 for the given text.
+ * @param text - Text the double SHA256 is computed for.
+ * @returns Hash as a 32-byte un-prefixed hex string.
+ */
+export function computeHash256(text: string): Hex {
```

Comment:
Worth making `text` input a `Hex`. We are interpreting it as `Buffer.from(text, "hex")` anyway.

Reply:
Done in 2b0b653d5d4131cc0c7b60bf2a42396a6af2eb0a.